contrib/gitdm/aliases

```diff
@@ -0,0 +1,148 @@
+Danny.Yates@mailonline.co.uk danny@codeaholics.org
```

Comment:
Does this serve the same purpose as our mailmap file? https://github.com/docker/docker/blob/master/.mailmap wondering if we can reuse that so that we don't have to maintain two files